src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetV1.java

```diff
@@ -1429,4 +1429,8 @@ public boolean isWrapperFor(Class<?> iface) throws SQLException {
       return false;
     }
   }
+
+  public SFBaseResultSet getSfBaseResultSet() {
```

Comment:
where do we use this?

Reply:
Oops, forgot to undo this change from the file.